spec/index.bs

```diff
@@ -16726,15 +16733,65 @@ expose real values whenever the feature is available on the adapter:
 - New WGSL extensions:
     - [=extension/subgroups=]
 
+<h3 id=texture-formats-tier1 data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"texture-formats-tier1"`
+</h3>
+
+Supports the below new {{GPUTextureFormat}}s with the {{GPUTextureUsage/RENDER_ATTACHMENT}},
+[=blendable=], `multisampling` capabilities and the {{GPUTextureUsage/STORAGE_BINDING}} capability
+with the {{GPUStorageTextureAccess/"read-only"}} and {{GPUStorageTextureAccess/"write-only"}}
+{{GPUStorageTextureAccess}}es:
+- {{GPUTextureFormat/"r16unorm"}}
+- {{GPUTextureFormat/"r16snorm"}}
+- {{GPUTextureFormat/"rg16unorm"}}
+- {{GPUTextureFormat/"rg16snorm"}}
+- {{GPUTextureFormat/"rgba16unorm"}}
+- {{GPUTextureFormat/"rgba16snorm"}}
+
+Allows the {{GPUTextureUsage/RENDER_ATTACHMENT}}, [=blendable=], `multisampling` and `resolve`
+capabilities on below {{GPUTextureFormat}}s:
+- {{GPUTextureFormat/"r8snorm"}}
+- {{GPUTextureFormat/"rg8snorm"}}
+- {{GPUTextureFormat/"rgba8snorm"}}
+
+Enables the extension {{GPUFeatureName/"rg11b10ufloat-renderable"}}.
+
+Allows the {{GPUStorageTextureAccess/"read-only"}} or {{GPUStorageTextureAccess/"write-only"}}
+{{GPUStorageTextureAccess}} on below {{GPUTextureFormat}}s:
+- {{GPUTextureFormat/"r8unorm"}}
+- {{GPUTextureFormat/"r8snorm"}}
+- {{GPUTextureFormat/"r8uint"}}
+- {{GPUTextureFormat/"r8sint"}}
+- {{GPUTextureFormat/"rg8unorm"}}
+- {{GPUTextureFormat/"rg8snorm"}}
+- {{GPUTextureFormat/"rg8uint"}}
+- {{GPUTextureFormat/"rg8sint"}}
+- {{GPUTextureFormat/"r16uint"}}
+- {{GPUTextureFormat/"r16sint"}}
+- {{GPUTextureFormat/"r16float"}}
+- {{GPUTextureFormat/"rg16uint"}}
+- {{GPUTextureFormat/"rg16sint"}}
+- {{GPUTextureFormat/"rg16float"}}
+- {{GPUTextureFormat/"rgb10a2uint"}}
+- {{GPUTextureFormat/"rgb10a2unorm"}}
+- {{GPUTextureFormat/"rg11b10ufloat"}}
+
+This feature adds no [=optional API surfaces=].
+
 # Appendices # {#appendices}
 
 ## Texture Format Capabilities ## {#texture-format-caps}
 
 ### Plain color formats ### {#plain-color-formats}
 
-All plain color formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}},
-and {{GPUTextureUsage/TEXTURE_BINDING}} usage. Additionally, all plain color formats support
-textures with {{GPUTextureDimension/"3d"}} dimension.
+A format that is <dfn dfn>enabled on the current device</dfn> means it is allowed to be used on the
+current device by default or with all the corresponding features enabled. The `Required Feature`
+column can either be empty that means the feature is [=enabled on the current device=] by default,
+or specify the feature required for the format to be [=enabled on the current device=].
+
+Any plain color format [=enabled on the current device=] supports {{GPUTextureUsage/COPY_SRC}},
+{{GPUTextureUsage/COPY_DST}}, and {{GPUTextureUsage/TEXTURE_BINDING}} usage. Additionally, any
+plain color formats [=enabled on the current device=] supports textures with
+{{GPUTextureDimension/"3d"}} dimension.
```

Comment:
We already have something like this definition in `Validate texture format required features` (the compressed formats already have this behavior).

I think we could simply keep the old text and add "supported":
```suggestion
All [$Validate texture format required features|supported$]
plain color formats support usages {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}},
and {{GPUTextureUsage/TEXTURE_BINDING}}, and dimension {{GPUTextureDimension/"3d"}}.
```